nevrending/foob
# Website Finishes + Fixes; Thanks @steelmaker86

* Create noises.html

* Update foob.html

* Update index.html

* Update index.html

* Update noises.html

* Update noises.html

* Update noises.html

* added placeholder background picture for noises

* Update noises.html

* Delete placeholder.png

* Add files via upload

* Add files via upload

* Update sounds.json

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update custom.css

* Update custom.css

* Update noises.js

* Update custom.css

* Added files and changed noises.js to fit them.

* Added changes to my own branch

Need to catch up and test

* Delete sounds.json

* Added changes in order to fit my own branch

Did this to test so I can mess with JQuery.

* changed to test with jquery

* Delete js directory

* Adding jquery to test

* Update noises.js

* Delete assets/audio directory

* Update noises.js

* Update custom.css

* Update noises.html

* Added stop button

* Added changes to noises.js and html

Removed the pause and play buttons. Only one audio file should play at a time now.

* Trying to fix background

Attempt to fix background of sound page

* Attempt to fix didn't work

* Testing out random

added random button to test out

* Fixed stop button

might want a shuffle button but for now that's irrelevant

* Adding community page

Also trying to fix the banner overlaying in noises.html

* Removed shrink to fit

* backtracked

* Added yosetti to main page

Just testing for now

* implementing yosetti to main page

* Fixed images

should be smaller

* Update index.html

* Update custom.css

* Update index.html

* Revert "Update index.html"

This reverts commit 769751f5131f0e1c886ed87774adf00dace0af5c.

* Adding in yosetti

* Attempting to make yosetti images cycle

* fixed tagging issue

* Making yosetti image proportional

* Update mainpage.js

* Update mainpage.js

* Trying to make video autoplay on mobile

* Update index.html

* want to see if changing video order helps

* It doesn't

* Update index.html

* Changing image rotation speed

* changed index and mainpage

* Update index.html

* undid changes

* Update bg-noises.png

* Begin working on community page

* Update sukon.html

* Update noises.html

* Update bg-sukon.png

* Worked on community page

* Update sukon.html

* Update sukon.html

* Update sukon.html

* updated sukon.html

* Update sukon.html

* Added community anchors

* Update sukon.html

* Testing something out

* changed sukon.html

* Update custom.css

* Update sukon.html

* Update custom.css

* changes

a

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Trying to fix issue on sukon title

* Update sukon.html

* Update sukon.html

* Update bg-sukon.png

* Revert "Update bg-sukon.png"

This reverts commit c4775a83dca513776fee13cc0a6ca5b4fa24bd36.

* Update bg-sukon.png

* Update bg-sukon.png

* Update bg-sukon.png

* Update bg-sukon.png

* Create mascots.html

* Update bg-sukon.png

* Wrote some mascot info

* MASCOTS

* Update mascots.html

* Added mascots to dropdown

* Update bg-noises.png

* Added community buttons

* Removed stuff from .github

* Create noises.html

* Update index.html

* Update index.html

* Update noises.html

* Update noises.html

* Update noises.html

* added placeholder background picture for noises

* Update noises.html

* Delete placeholder.png

* Add files via upload

* Add files via upload

* Update sounds.json

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update noises.js

* Update custom.css

* Update custom.css

* Update noises.js

* Update custom.css

* Added files and changed noises.js to fit them.

* Added changes to my own branch

Need to catch up and test

* Adding jquery to test

* Added changes in order to fit my own branch

Did this to test so I can mess with JQuery.

* changed to test with jquery

* Delete js directory

* Update noises.js

* Update noises.js

* Update custom.css

* Update noises.html

* Added stop button

* Added changes to noises.js and html

Removed the pause and play buttons. Only one audio file should play at a time now.

* Trying to fix background

Attempt to fix background of sound page

* Attempt to fix didn't work

* Testing out random

added random button to test out

* Fixed stop button

might want a shuffle button but for now that's irrelevant

* Adding community page

Also trying to fix the banner overlaying in noises.html

* Removed shrink to fit

* backtracked

* Added yosetti to main page

Just testing for now

* implementing yosetti to main page

* Fixed images

should be smaller

* Update index.html

* Update custom.css

* Update index.html

* Revert "Update index.html"

This reverts commit 769751f5131f0e1c886ed87774adf00dace0af5c.

* Adding in yosetti

* Attempting to make yosetti images cycle

* fixed tagging issue

* Making yosetti image proportional

* Update mainpage.js

* Update mainpage.js

* Trying to make video autoplay on mobile

* Update index.html

* want to see if changing video order helps

* It doesn't

* Update index.html

* Changing image rotation speed

* changed index and mainpage

* Update index.html

* undid changes

* Update bg-noises.png

* Begin working on community page

* Update sukon.html

* Update noises.html

* Update bg-sukon.png

* Worked on community page

* Update sukon.html

* Update sukon.html

* Update sukon.html

* updated sukon.html

* Update sukon.html

* Added community anchors

* Update sukon.html

* Testing something out

* changed sukon.html

* Update custom.css

* Update sukon.html

* Update custom.css

* changes

a

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Update sukon.html

* Trying to fix issue on sukon title

* Update sukon.html

* Update sukon.html

* Update bg-sukon.png

* Revert "Update bg-sukon.png"

This reverts commit c4775a83dca513776fee13cc0a6ca5b4fa24bd36.

* Update bg-sukon.png

* Update bg-sukon.png

* Update bg-sukon.png

* Update bg-sukon.png

* Create mascots.html

* Update bg-sukon.png

* Wrote some mascot info

* MASCOTS

* Update mascots.html

* Added mascots to dropdown

* Update bg-noises.png

* Added community buttons

* Removed stuff from .github

* Removed from1st announcement

* Update sukon.html

* Update sukon.html

* Update index.html

* Added art canvas to the index

* Update noises.js
